quantum/template/avr/rules.mk

```
@@ -27,6 +27,6 @@ BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 MIDI_ENABLE = no            # MIDI support
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-AUDIO_ENABLE = no           # Audio output on port C6
+AUDIO_ENABLE = no           # Audio output (needs a pin configured in config.h, see docs/feature_audio.md)
```

```suggestion
AUDIO_ENABLE = no           # Audio output
```
IMO this is unnecessary. The comment on `BACKLIGHT_ENABLE` does not point to the docs either; compilation should fail if a pin is not specified.

---

ok - upstream/master does not do this though... PR #6165 (currently) only warns on missing pins (to avoid the breaking change process)